app/services/themes/create.rb

```diff
@@ -0,0 +1,84 @@
+# frozen_string_literal: true
+
+class Themes::Create
+  include Service::Base
+
+  params do
+    attribute :name, :string
+    attribute :user_id, :integer
+    attribute :user_selectable, :boolean, default: false
+    attribute :color_scheme_id, :integer
+    attribute :component, :boolean, default: false
+    attribute :theme_fields, :array
+    attribute :default, :boolean, default: false
+  end
+
```

Comment:
I wasn't sure on this, IDK how common it is to interleave policies with steps so I wanted to make it clearer...but also totally happy to do away with it too :)

Reply:
Well, policies are steps too :grin: I get that usually we try to group DSL keywords together, but in the case of services, I think it’s better to group all the steps together because they’re actually describing the execution flow, so they’re like one block. I don’t know if I’m making a lot of sense :sweat_smile:

Reply:
Totally makes sense, I know policies are special steps too! Will just remove the spaces :+1: